service/storage_service.hh

```diff
@@ -213,6 +213,10 @@ public:
         return _db;
     }
 
+    shared_ptr<distributed<cql_transport::cql_server>> get_dist_cql_server() const {
```

Comment:
Where is this used? I couldn't find the place.

Reply:
aaah, a ghost from virtual tables. Removed